data/css/blog.scss

```diff
@@ -122,8 +122,9 @@ $highlighted-background-color: darken($background-dark-color, 5%) !default;
 
 .article-page {
   .related-posts {
-    margin: 0px 50px 20px 50px;
-
+    background-color: $background-light-color;
+    margin-right: 15px;
+    padding: 0px 50px 20px 50px;
```

Comment:
@ptomato let me try to sketch it here, hehe:

1) this change is about adding a background color to the Related Articles content goup.
2) previously, we had that margin applied to content group, so when adding a background color, the background would only extend to immediate boundaries of the content goup (looking like a floating box on top of the article background color).
3) so, to fix that I switch to padding so we would have the spacing around the content but extending its background on top of the articles background.
4) lastly, the final margin-right is simply to prevent the content group to hide the scrolled window slider.

does this make sense?